src/channel.ts

```diff
@@ -172,6 +193,90 @@ export class Channel {
     });
   };
 
+  /**
+   * For cross channel.
+   *
+   * transactions(() => {
+   *   const useResult = channel.whatever();
+   *
+   *   useResult.something();
+   *
+   *   channel.another();
+   * }, 'retry');
+   */
+  static transaction = async (
+    fn: () => Promise<RequestResult>,
```

Comment:
```suggestion
    tx: () => Promise<RequestResult>,
```

and elsewhere. `fn` is pretty undescriptive

Reply:
I don't think `tx` is more descriptive. I'll add a JSDoc?

Reply:
yeah sure